README.md

```diff
@@ -423,24 +411,31 @@ To solve this error You must go to [hoyolab.com](https://www.hoyolab.com/genshin
 
 Tip for developers: the first 682 characters (85 bytes) of the authkey are always the same for each user. You can use this to easily indentify if an authkey belongs to the same user as another authkey.
 
+Element | Access level | Sensitivity
+|--|--|--|
+`UID` | Read access to your Battle Chronicle's public data | Safe
+`ltuid` + `ltoken` | Read access to your full Battle Chronicle (characters, abyss clears, ...) + few actions possible (claim daily reward) | Quite safe
+`authkey` | 24h access to your wish history, top-up history and other logs... | Safe
+`account_id` + `cookie_token` | Full Battle Chronicle + can be connected as you on most mihoyo.com websites | Don't share those
+
 ## How can I claim daily rewards for other users.
 When making projects that claim daily rewards for other users you can pass in a `cookie` parameter - `claim_daily_rewards(cookie={'ltuid': ..., 'ltoken': ...})`.
 This will avoid having to set global cookies so you can use it with threading.
 
-This cookie parameter is avalible for the majority of other accounts.
+This cookie parameter is available for the majority of other accounts.
 
 ## Is it possible that my account can be stolen when I login with the cookie?
-I would like to be completely clear in this aspect, I do no have any way to access the cookies you use to login. If you give your cookie to someone it is indeed possible to get into your account but that doesn't yet mean they can do anything with it. The most probable thing a hacker would do is just do a password request, but since version `1.3` they will need to confirm this request with an email. That means they would need to know what your email is and have a way to get into it, which I doubt they can. Since version `1.5` there is also 2FA which will make it completely impossible to steal your account.
+I would like to be completely clear in this aspect, I do no have any way to access the cookies you use to login. Having your cookie won't give an access to your account, except for `cookie_token` and `account_id` which allow a partial access (web events and mihoyo's website) but not your personal data / reset password. Plus, since version `1.5` there is also 2FA which makes it completely impossible to steal your account.
 
-They can of course access your data like email, phone number and real name, however those are censored so unless they already have an idea what those could be that data is useless to them. (For example the email may be `thesadru@gmail.com` but it'll only show up as `th****ru@gmail.com`)
+No personal data is accessible except your hoyolab nickname and email (partially censored, for example `thesadru@gmail.com` would only show up as `th****ru@gmail.com`).
 
-TL;DR unless you have also given your password away your account cannnot be stolen.
+TL;DR unless you have also given your password away your account cannot be stolen.
```

Comment:
this sentence now implies something else due to the changes. While your account may not be taken away from you and it's not possible to actually login in-game it doesn't mean it cannot be manipulated through mihoyo's API.

Reply:
No sure of what you mean, are you thinking of some API features that aren't implemented in genshinstats? What could be done?

Reply:
I do not implement any features that are not related to statistics. Anything forum or account management-related will never be implemented. That's just not what I want to have this lib become.